bundle/EventListener/ViewTaggerListener.php

```diff
@@ -23,10 +26,10 @@ public function __construct(ResponseTagger $responseTagger)
 
     public static function getSubscribedEvents(): array
     {
-        return [SiteApiEvents::RENDER_CONTENT => 'onRenderContent'];
+        return [Events::NG_VIEW_CONTENT_RENDER => 'onRenderView'];
```

Comment:
I feel that this should be kept in sync (as Symfony did in 4.3), e.g.

`return [Events::RENDER_VIEW` => 'onRenderView'];`

Btw, the reason why I named it `RenderContentEvent` and `::RENDER_CONTENT` is because you as a developer actually render content and not the view from the Twig templates :)

Reply:
Renamed constant and method in https://github.com/netgen/ezplatform-site-api/pull/146/commits/0f594480db1c0edc7d8c82371f64a6e1302f9f48.

> Btw, the reason why I named it RenderContentEvent and ::RENDER_CONTENT is because you as a developer actually render content and not the view from the Twig templates :)

OK, but the view is what is rendered and the event object can be generic as it's not used for dispatch. So it might make sense to have it this way? :)

Reply:
How about just `RENDER_VIEW` then? `NG_VIEW_CONTENT_RENDER` sounds overly complicated :)

Reply:
As discussed in person, Symfony changed event dispatching in 4.3, so I'm going to update this accordingly. (See https://symfony.com/blog/new-in-symfony-4-3-simpler-event-dispatching)